scene/gui/slider.cpp

```diff
@@ -269,98 +269,121 @@ void Slider::_notification(int p_what) {
+
+				if (symmetric_fill && get_min() < 0.0 && get_max() > 0.0 && !Math::is_equal_approx(get_min(), get_max())) {
+					const double zero_ratio = get_min() / (get_min() - get_max());
+					int initial_pos = Math::round(areasize * (1 - zero_ratio) + grabber->get_height() / 2 - grabber_shift);
+					int grabber_pos = Math::round(areasize * (1 - ratio) + grabber->get_height() / 2 - grabber_shift);
+
+					if (grabber_pos < initial_pos) {
+						SWAP(grabber_pos, initial_pos);
+					}
+
+					grabber_area->draw(ci, Rect2i(Point2i((size.width - widget_width) / 2, initial_pos), Size2i(widget_width, grabber_pos - initial_pos)));
+				} else {
+					int grabber_pos = Math::round(areasize * ratio + grabber->get_height() / 2 - grabber_shift);
+
+					grabber_area->draw(ci, Rect2i(Point2i((size.width - widget_width) / 2, size.height - grabber_pos), Size2i(widget_width, grabber_pos)));
+				}
 
 				if (ticks > 1) {
-					int grabber_offset = (grabber->get_height() / 2 - tick->get_height() / 2);
+					const int grabber_offset = (grabber->get_height() / 2 - tick->get_height() / 2);
 					for (int i = 0; i < ticks; i++) {
 						if (!ticks_on_borders && (i == 0 || i + 1 == ticks)) {
 							continue;
 						}
 						int ofs = (i * areasize / (ticks - 1)) + grabber_offset - grabber_shift;
 
-						if (ticks_position == TICK_POSITION_BOTTOM_RIGHT || ticks_position == TICK_POSITION_BOTH) {
-							tick->draw(ci, Point2i(widget_width + (size.width - widget_width) / 2 + theme_cache.tick_offset, ofs));
-						}
-
-						if (ticks_position == TICK_POSITION_TOP_LEFT || ticks_position == TICK_POSITION_BOTH) {
-							Point2i pos = Point2i((size.width - widget_width) / 2 - tick->get_width() - theme_cache.tick_offset, ofs);
-							tick->draw_rect(ci, Rect2i(pos, Size2i(-tick->get_width(), tick->get_height())));
-						}
-
 						if (ticks_position == TICK_POSITION_CENTER) {
 							tick->draw(ci, Point2i((size.width - tick->get_width()) / 2 + theme_cache.tick_offset, ofs));
+						} else {
+							if (ticks_position == TICK_POSITION_BOTTOM_RIGHT || ticks_position == TICK_POSITION_BOTH) {
+								tick->draw(ci, Point2i(widget_width + (size.width - widget_width) / 2 + theme_cache.tick_offset, ofs));
+							}
+
+							if (ticks_position == TICK_POSITION_TOP_LEFT || ticks_position == TICK_POSITION_BOTH) {
+								Point2i pos = Point2i((size.width - widget_width) / 2 - tick->get_width() - theme_cache.tick_offset, ofs);
+								tick->draw_rect(ci, Rect2i(pos, Size2i(-tick->get_width(), tick->get_height())));
+							}
 						}
 					}
 				}
-				grabber->draw(ci, Point2i(size.width / 2 - grabber->get_width() / 2 + theme_cache.grabber_offset, size.height - ratio * areasize - grabber->get_height() + grabber_shift));
+				if (grabber.is_valid()) {
```

Comment:
I wonder if this will be useful when users uses empty textures and draws their own custom grabbers, like this

<img width="870" height="518" alt="image" src="https://github.com/user-attachments/assets/4a0e9cc3-4dd5-4fba-8b8a-b28cd9747b36" />